format/Flight.proto

```diff
@@ -287,7 +287,9 @@ message FlightData {
 
   /*
    * The actual batch of Arrow data. Preferably handled with minimal-copies
-   * coming last in the definition to help with sidecar patterns.
+   * coming last in the definition to help with sidecar patterns (it is
+   * expected that some implementations will parse this field off the wire
```

Comment:
I would replace "parse" with "fetch". Part of the intent is _not_ have to parse at all.
Of course, the way we do it currently is not exactly idiomatic.

Reply:
Done.